### PR TITLE
[CFM-1141] Project content type fields addition (6 fields), removal (1 field)1

### PR DESCRIPTION
**Pull request by anvmn**
Parameters described at CFM 1140 comments.

### CODE REVIEW

**Comment by anvmn** [comment]
https://issuetracker.amplexor.com/jira/browse/CFM-1140

selection_068

**Comment by anvmn** [comment]
@or Travis failed as one behat test, dealing with search API failed to complete.
It has no relation to Project content type.

**Comment by ordavidil** [comment]
@anvmn 
Did you check this specific test on your local machine?
please do so, and try to understand the problem.

**Comment by ordavidil** [comment]
@anvmn 
This PR is actually for sub-task CFM-1141 and not the whole issue CFM-1140, right ?

**Comment by anvmn** [comment]
@ordavidil 
  Yes, for CFM 1141. I edited PR title to reflect it.

**Comment by ordavidil** [comment]
Thanks,

can you please also set the links between GitHub and JIRA ?

**Comment by anvmn** [comment]
@ordavidil 
Updated the PR with link to CFM.
Saw you added the link to PR at CFM yourself.

**Comment by anvmn** [comment]
@ordavidil 
Regarding failing behat test, I ran the test on it's ownd and all @api tests locally, and in both cases it completed fine.
Seems like travis issue.

selection_075

File: project/profiles/capacity4more/modules/c4m/demo/c4m_demo_content/csv/node/project.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+project_2,user_1,Human Genome Project,human-genome,dna.png,dna-banner.jpg,1990-01-01 00:00:00;2003-12-05 00:00:00,16000000,EUR,Qatari Diar Real Estate Investment Company,,CRIS-1990-001,CRIS 000199001010003,"The Human Genome Project (HGP) is an international scientific research project with the goal of determining the sequence of chemical base pairs which make up human DNA, and of identifying and mapping all of the genes of the human genome from both a physical and functional standpoint. It remains the world's largest collaborative biological project. The project was proposed and funded by the US government ; planning started in 1984, the project got underway in 1990, and was declared complete in 2003. A parallel project was conducted outside of government by the Celera Corporation, or Celera Genomics, which was formally launched in 1998. Most of the government-sponsored sequencing was performed in universities and research centres in China, France, Germany, Japan, Spain, the United Kingdom, and the United States",t_wind,Worldwide,group_5,project_1,1,0
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/demo/c4m_demo_content/csv/node/project.csv]: @anvmn 
I think you should remove one of the `,` after the `Qatari Diar Real Estate Investment Company`

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/demo/c4m_demo_content/csv/node/project.csv]: forget about it

**Comment by ordavidil** [comment]
@anvmn 
now that we know that `implemented_by` is for sure the one to make troubles over here, please check again the behat tests.

**Comment by ordavidil** [comment]
re-roll

**Comment by anvmn** [comment]
Closed in favor of #1131

**Pull request closed without merging**